templates/cisco_nxos_show_route-map.template

```diff
@@ -0,0 +1,28 @@
+Value Required NAME (\S+)
+Value Required ACTION (\S+)
+Value Required SEQ (\d+)
+Value List MATCH_CLAUSES (.+?)
+Value List SET_CLAUSES (.+?)
+
+Start
+  ^route-map\s+${NAME},\s+${ACTION},\s+sequence\s+${SEQ}\s*$$
+  ^\s+Match\s+clauses -> Match
+  ^\s+Set\s+clauses -> Set
+  ^.+ -> Error
```

Comment:
Do you mind changing the `Error` lines out for the following:
```
  ^\s*$$
  ^. -> Error
```

After that, everything else looks good to merge. Just wanting to enforce our standards on new templates.

Reply:
ack, updated.